### PR TITLE
refactoring execution test pipeline

### PR DESCRIPTION
**Pull request by saurabh-mridul**
creating execution test template to avoid steps duplication for every region
renaming the pipelines as per standard 
renaming the folder to pipelines

### CODE REVIEW

File: Pipelines/ExecuteInAKS.ps1 [hunk 1]
```diff
@@ -1,98 +1,98 @@
...
-    New-Item $AKSClusterName  -ItemType "directory"
-
-    kubectl cp $Namespace/${masternode}:report/ $AKSClusterName --context $currentcontext
-
-    kubectl delete namespace $Namespace --context $currentcontext
-
-  }
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$True)]
```
Comment (agrawalamit31) [inline Pipelines/ExecuteInAKS.ps1:3]: format indentation

File: Pipelines/ExecuteInAKS.ps1 [hunk 2]
```diff
@@ -1,98 +1,98 @@
...
+
+for ($counter=0; $counter -lt $AKSClusterNames.Count; $counter++){
+
+    az login --service-principal   --username  $ClientIds[$counter]  --password $ClientSecrets[$counter]   --tenant $TenantId
+
+    az aks get-credentials --resource-group $ResourceGroups[$counter] --name $AKSClusterNames[$counter] --overwrite --admin
+}
+
+ $AKSClusterNames | ForEach-Object { 
+
```
Comment (agrawalamit31) [inline Pipelines/ExecuteInAKS.ps1:47]: remove extra line break

File: Pipelines/test-execution-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
+    type: string
+    default: JMeterFiles
+
+  - name: JmeterFileName
```
Comment (agrawalamit31) [inline Pipelines/test-execution-pipeline.yml]: rename JmeterFileName to JMeterFileName

File: Pipelines/test-execution-pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,102 @@
+parameters:
+  - name: IsMultiRegionEnabled
+    type: boolean
+    default: false
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
```
Comment (agrawalamit31) [inline Pipelines/test-execution-pipeline.yml]: rename JmeterFolderPath to JMeterFolderPath

File: Pipelines/test-execution-steps.yml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    default: false
+
+  - name: RegionNumber
+    type: number
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
```
Comment (agrawalamit31) [inline Pipelines/test-execution-steps.yml]: rename JmeterFolderPath to JMeterFolderPath

File: Pipelines/test-execution-steps.yml [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    type: string
+    default: JMeterFiles
+
+  - name: JmeterFileName
```
Comment (agrawalamit31) [inline Pipelines/test-execution-steps.yml]: rename JmeterFileName to JMeterFileName

File: Pipelines/test-execution-steps.yml [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+  - name: Namespace
+    type: string
+
+  - name: ServiceConnection
+    type: string
+
+  - name: KeyVaultName
+    type: string
+
+  - name: SecretNames
```
Comment (agrawalamit31) [inline Pipelines/test-execution-steps.yml:49]: can we keep it as SecretName ? Is it array ?

File: Pipelines/test-execution-steps.yml [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+  - name: AKSSPNClientId
+    type: string
+
+  - name: PerfTestResourceId
+    type: string
+
+  - name: PerfTestClientId
+    type: string
+
+  - name: CSVFileNames
```
Comment (agrawalamit31) [inline Pipelines/test-execution-steps.yml:70]: are we passing multiple files here ?

File: Pipelines/test-execution-pipeline.yml [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+  - name: Loops
+    type: number
+    default: 3
+
+  - name: RampUpTime
+    type: number
+    default: 10
+
+variables:
+  # Tenant: ""
```
Comment (agrawalamit31) [inline Pipelines/test-execution-pipeline.yml:35]: I forgot why did we keep commented varaiable here

**Review by agrawalamit31 — CHANGES_REQUESTED** [review]

**Review by Shuvajyoti — APPROVED** [review]
Ensure existing functionality not broken with integration test run.

File: Pipelines/test-execution-pipeline.yml (hunk 9 above)
Comment (saurabh-mridul) [reply Pipelines/test-execution-pipeline.yml:35]: Its added for reference so users have two ways of adding values for variables - 
1. they can directly edit the yml file and uncomment variables and pass values
2. refer variables names from here and pass during the run

File: Pipelines/test-execution-steps.yml (hunk 8 above)
Comment (saurabh-mridul) [reply Pipelines/test-execution-steps.yml:70]: not sure, we could.. it's the existing functionality

File: Pipelines/test-execution-steps.yml (hunk 7 above)
Comment (saurabh-mridul) [reply Pipelines/test-execution-steps.yml:49]: yes its an array, you can pass comma separated values

File: Pipelines/ExecuteInAKS.ps1 (hunk 1 above)
Comment (saurabh-mridul) [reply Pipelines/ExecuteInAKS.ps1:3]: Let's not modify this script, we need to discuss with partner teams on all the suggestions we have identified. will fix in in follow up PR.

File: Pipelines/test-execution-pipeline.yml (hunk 4 above)
Comment (saurabh-mridul) [reply Pipelines/test-execution-pipeline.yml]: updated the names

**Review by agrawalamit31 — APPROVED** [review]
@saurabh-mridul  - Please resolve all conversation

**Pull request merged**